packages/devtools_app/lib/src/ui/icons.dart

```diff
@@ -157,6 +157,16 @@ class FlutterIcons {
   static const DevToolsIcon debugBanner = UrlIcon('/icons/debug_banner.png');
   static const DevToolsIcon history = UrlIcon('/icons/history.png');
 
+  static const DevToolsIcon pause = MaterialIcon('pause', contrastForeground);
```

Comment:
as you are only using these icons from the Flutter code, use the regular Flutter icons instead of this weird way of defining icons to play nicely with the dart:html version of the app.

Reply:
done.